crates/nu-command/src/filesystem/save.rs

```diff
@@ -349,7 +349,11 @@ fn stream_to_file(
     span: Span,
     progress: bool,
 ) -> Result<PipelineData, ShellError> {
-    let mut writer = BufWriter::new(file);
+    // Note: when saved from external stream to file, no need to use BufWriter
+    // so we can see the output of external stream in a file immediately.
+    //
+    // It's good for users to see running status of external program.
```

Comment:
i'm personnaly not a huge fan of comments in code, they can be hard to maintain and evolve with the code... :confused: 

maybe we can just link to this PR, which already contains this information? :yum: 

in the end, i do not have a super strong opinion on this, as this change is not super trivial :wink:

Reply:
Thanks, I think you're right :-D  Adjusted the comment

Reply:
cool :yum: